### ISSUE TITLE
ListBox.DisplayMember property should be used as an accessible name for combobox items

### ISSUE THREAD

**Issue by Zheng-Li01** [dotnet/winforms#3706]
* .NET Core Version: 
.NET SDK 5.0.100.rc1.20405.13

* Have you experienced this same bug with .NET Framework?: 
No

* More info:
1. The issue also can reproduce .NET SDK 3.1.
2. Tested ListBox with items from non-data binding and data binding, the Name of list items are displayed correctly. 

**Problem description:**
AT user does not know what items are available for selection

Minimal repro:
1. Open the attached project.
[Project.zip](https://github.com/dotnet/winforms/files/5040304/Project.zip)
2. Build & Run the application under the inspect tool.

**Expected behavior:**
Accessible tree should show the same strings as the drop down shows
![Framework](https://user-images.githubusercontent.com/38325459/89628256-9e5d5200-d8ce-11ea-8eca-049d8ce97cd2.PNG)

**Actual behavior:**
type name is used as accessible name for the ListBox list items, because object.ToString is called:
![Core](https://user-images.githubusercontent.com/38325459/89628279-a74e2380-d8ce-11ea-9870-2454456d5049.PNG)

**Comment by Tanya-Solyanik** [comment]
Only the combobox was fixed previously - https://github.com/dotnet/winforms/pull/3550.
@merriemcgaw - this repros in 3.1, should we fix it there? I does not repro in 4.8

**Comment by merriemcgaw** [comment]
Nah, I don't think this needs to be serviced. But we do want to try to get it in to 5.0 RC2. Do you see any issues with that?

**Comment by Zheng-Li01** [comment]
Verified the issue with latest .NET 5.0.100-rc2.20459.18, the isssue has been fixed that items' name displayed for now.
![CoreIssue](https://user-images.githubusercontent.com/38325459/92696660-c3852a80-f37c-11ea-952f-469d9c1492f8.PNG)
